src/main/java/ch/njol/skript/expressions/ExprCommand.java

```diff
@@ -79,6 +80,8 @@ protected String[] get(final Event e) {
 			s = ((PlayerCommandPreprocessEvent) e).getMessage().substring(1).trim();
 		} else if (e instanceof ServerCommandEvent) {
 			s = ((ServerCommandEvent) e).getCommand().trim();
+		} else if (e instanceof UnknownCommandEvent) {
+			s = ((UnknownCommandEvent) e).getCommandLine().trim();
```

Comment:
same issue as previous review, could be sorted out with `SUPPORTS_UNKNOWN_COMMAND && e instance of`